CreeDictionary/tests/conftest.py

```diff
@@ -14,7 +14,7 @@
 
 from DatabaseManager.xml_importer import import_xmls
 
-settings.register_profile("default", deadline=timedelta(milliseconds=1000))
+settings.register_profile("default", deadline=timedelta(milliseconds=2000))
```

Comment:
`test_search_for_exact_lemma` took 1500 ms in a run and `hypothesis` threw `DeadlineExceededError`.

Note it's not necessarily our searching is too slow. As commented before, the `random_lemma` custom strategy draws lemma in an inefficient way, causing the function to run for longer time.

Reply:
:( sounds like time for future improvement.